skimage/transform/tests/test_thin_plate_splines.py

```diff
@@ -0,0 +1,242 @@
+import numpy as np
+import pytest
+
+import skimage as ski
```

Comment:
This is not standard practice as of yet. @lagru do we want to include this import in all test files moving forward?

Reply:
I think I just used what is convenient. :)